aeneas/src/mach/GlobalRegAlloc.v3

```diff
@@ -217,15 +211,15 @@ class GlobalRegAlloc(regSet: MachRegSet, codegen: SsaMachGen) {
 			}
 		}
 	}
-	def allocDefReg(vreg: VReg, constraint: int) -> int {
+	def allocDefReg(vreg: VReg, constraint: int, restore: bool) -> int {
```

Comment:
Should we name the new param "genSave"? Because AFAICT that is the behavior that it controls here.